changelogs/changelog.yaml

```diff
@@ -815,6 +815,10 @@ releases:
       name: netbox_user_group
       namespace: ''
     release_date: '2024-09-12'
+  3.20.1:
+    changes:
+      minor_changes:
+        - Add example for using ansible variables in lookup (https://github.com/netbox-community/ansible_modules/pull/1332)
```

Comment:
This needs to be created as a file in the `fragments` directory

Reply:
Thanks! I have added a fragment. Hope it looks better now :)